trigger/project_triggers.go

```diff
@@ -78,7 +78,7 @@ func TriggerDownstreamVersion(ctx context.Context, args ProcessorArgs) (*model.V
 			if args.TriggerType == model.ProjectTriggerLevelPush {
```

Comment:
GH doesn't let me comment on unchanged lines, but just to double check, is the if statement on L77 still supposed to check the upstream project?

Reply:
Yes, because we're looping through the downstream project's modules and checking if the upstream project is one of the downstream project's modules.